website/docs/r/sqs_queue_policy.html.markdown

```diff
@@ -8,11 +8,14 @@ description: |-
 
 # Resource: aws_sqs_queue_policy
 
-Allows you to set a policy of an SQS Queue
-while referencing ARN of the queue within the policy.
+Allows you to set a policy of an SQS Queue while referencing the ARN of the queue within the policy.
+
+~> **Note:** AWS will hang indefinitely when creating or updating an [`aws_sqs_queue`](/docs/providers/aws/r/sqs_queue.html) with an associated policy if `Version = "2012-10-17"` is not explicitly set in the policy. [See below](#timeout-problems-creatingupdating) for an example of how to avoid this issue.
```

Comment:
```suggestion
!> AWS will hang indefinitely when creating or updating an [`aws_sqs_queue`](/docs/providers/aws/r/sqs_queue.html) with an associated policy if `Version = "2012-10-17"` is not explicitly set in the policy. [See below](#timeout-problems-creatingupdating) for an example of how to avoid this issue.
```